dashboard/src/components/ChartList/ChartListItem.tsx

```diff
@@ -1,9 +1,8 @@
 import * as React from "react";
-import { Link } from "react-router-dom";
 
 import placeholder from "../../placeholder.png";
 import { IChart } from "../../shared/types";
-import Card, { CardContent, CardIcon } from "../Card";
+import { PreformattedCard } from "../Card";
```

Comment:
I am not convinced about the naming `PreformattedCard`. Why preformatted? Maybe something that represents its behavior or purpose like `genericCard`, `infoCard`?

Reply:
I don't have a strong opinion about the name, I just wanted to emphasize that it has a specific layout. `infoCard` works for me as well.